src/test/java/com/znsio/rpap/WebExampleTest.java

```diff
@@ -0,0 +1,80 @@
+package com.znsio.rpap;
+
+import com.applitools.eyes.selenium.fluent.Target;
+import com.epam.reportportal.annotations.Step;
+import com.znsio.rpap.pages.WebExample;
+import com.znsio.rpap.utils.JsonDataManager;
+import com.znsio.rpap.utils.JsonDataProvider;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+import static com.znsio.rpi.utils.ReportPortalLogger.captureAndAttachScreenshot;
+
+
+public class WebExampleTest extends BaseTest {
```

Comment:
We are going to use a listener instead of inheritance, right?

Reply:
This is listed in the **Upcoming Changes** section of the PR description and will be part of follow-up PR. Setting aside some work Raghav/Megha

Reply:
Add a TODO so that we do not miss this

Reply:
Added